### PR TITLE
Change entity bounding-box per action instead of every action

### PR DESCRIPTION
**Pull request by kokosha**
This is a small change in logic that fix the rendering of monster that have some action with underground sprites, and not let the error to affect the other action.

### CODE REVIEW

**Comment by hasenbanck** [comment]
Should we leave this open as a draft until you found the correct math?

**Comment by kokosha** [comment]
This is a small change in logic that fix the rendering of monster that have some action with underground sprites, and not let the error to affect the other action.
Maybe the comment is not well-explained.
The underground sprite is going to be fixed in another pull request, when I get the math correct.

File: korangar/src/loaders/animation/mod.rs [hunk 1]
```diff
@@ -201,63 +201,62 @@ impl AnimationLoader {
...
+            // To resolve this, we resized the frames to ensure
+            // identical frame size and origin point at (0,0) between frames.
+
+            // First, identify the bounding box that encompasses all motion
+            // frames.
+            let mut min_top = i32::MAX;
+            let mut max_bottom = 0;
+            let mut min_left = i32::MAX;
+            let mut max_right = 0;
+            // The player can change the sprite and causes an offset from a image to
```
Comment (hasenbanck) [inline korangar/src/loaders/animation/mod.rs:218]: Can you please add a empty line before the comment?

**Comment by kokosha** [comment]
I think it is ok

**Pull request merged**